packages/tpetra/core/test/MultiVector/Transform.cpp

```diff
@@ -270,6 +273,7 @@ namespace { // (anonymous)
     constexpr double flagValue = -1.0;
     X.putScalar (flagValue);
     Y.putScalar (666.0);
+    Kokkos::fence();
```

Comment:
See above; we should be able to remove this fence.

Reply:
This fence was apparently a mistake.